src/components/header/header-component.vue

```diff
@@ -0,0 +1,60 @@
+<style lang="scss">
```

Comment:
for cleanliness you may want to break styles out into a separate scss file and `@import` it here. 

Same for `menu-component.vue`

Reply:
Thanks! Done 👍